test/JsBuiltIns/ChakraLib.js

```diff
@@ -0,0 +1,17 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+    {
+        name: "Access to __chakraLibrary work",
+        body: function () {
+            assert.isNotUndefined(__chakraLibrary, "__chakraLibrary must NOT be undefined as we're using the -LdChakraLib flag.");

```

Comment:
>__chakraLibrary [](start = 34, length = 15)

What if you try to overwrite this like you did in the other test? You should ensure that inside of the JsBuiltIn code it is still the correct object.